openIMIS/developer_tools/skeletons.py

```diff
@@ -0,0 +1,56 @@
+import os
+
+
+def get_skeleton_setup(module_name, author, author_email):
+    return \
+        F"import os{os.linesep}" \
```

Comment:
I suggest to have the skeleton code in actual file and then you read the file and replace the module name and other information. I will be easier to manage and update later.

Reply:
ok sure. Do you see in such way as:
- catalog 'skeletons'
    - setup.py
    - README.md
    - urls.py
    - ...
    - etc?
Or do you see this in another way? Let me know.

Reply:
I added skeletons as a templates.